### PR TITLE
Add Opal C1 0.26

### PR DESCRIPTION
**Pull request by nicholaschiang**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/opal.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+cask "opal" do
```
Comment (bevanjkay) [inline Casks/opal.rb:1]: I wonder if `opal-camera` or the like is a better token here?
Interested in your thoughts @nicholaschiang

Comment (nicholaschiang) [reply Casks/opal.rb:1]: i'm fine with either one. i don't think `opal` has a cask yet, but i guess the `opal` formulae does exist...

Comment (bevanjkay) [reply Casks/opal.rb]: The other option would be `opal-webcam-software` - whichever we go with the filename will need to match the token.

Comment (nicholaschiang) [reply Casks/opal.rb:1]: the installed application name is just "Opal" so i think it makes sense to also name this cask `opal`.

**Comment by miccal** [comment]
Thank you @nicholaschiang.

**Pull request merged**